plugin/evm/rawdb/accessors_state_sync_test.go

```diff
@@ -7,13 +7,13 @@ import (
 	"testing"
 
 	"github.com/ava-labs/libevm/common"
-	"github.com/ava-labs/libevm/core/rawdb"
+	ethrawdb "github.com/ava-labs/libevm/core/rawdb"
 	"github.com/stretchr/testify/require"
 )
 
 func TestClearPrefix(t *testing.T) {
 	require := require.New(t)
-	db := rawdb.NewMemoryDatabase()
+	db := ethrawdb.NewMemoryDatabase()
```

Comment:
not sure why the rename to ethrawdb is needed here?

Reply:
For consistency, given it's used as ethrawdb in other places in plugin/evm/rawdb. Also to remove the confusion given the package is also named `rawdb`.

Reply:
This also makes me think, it might be interesting to rename plugin/evm/rawdb package name from `rawdb` to `customrawdb`, and then use `rawdb` instead of `ethrawdb`? So that it fits with other source files? Thoughts?

Reply:
that works for me, overall I think the "extra"/"custom" packages should follow a single naming convention (custom seems fine) and live under `plugin/evm`

Reply:
Done in #888 so that file movements are tracked properly and not mixed up with changes done here.

Reply:
Also, regarding this, should we rename `_ext.go` files to `_custom.go` files? Or it's fine as is? 🤔